ui/src/api/base.js

```diff
@@ -10,6 +10,13 @@ const apiConfig = {
   withCredentials: true,
 };
 
+const adminConfig = {
```

Comment:
I would have gone with something like:
```
const invenioBaseURL = process.env.NODE_ENV === 'production' ? '/' : `${process.env.REACT_APP_BACKEND_DEV_BASE_URL}/
const apiConfig = {
  baseURL: invenioBaseURL,
  withCredentials: true,
}
```
to avoid duplication, and then append the `/admin` in the route file, but I am also fine with this :)